bot.py

```diff
@@ -310,44 +337,80 @@ async def play(self, message, command_content):
         else:
             self.next_in_queue()
 
-    async def stop(self, _message, _command_content):
+    async def stop(self, message, _command_content):
         """
         Stop currently playing song
         """
-        if self.voice_client:
-            self.voice_client.stop()
+        if not self.voice_client:
```

Comment:
This is at the beginning of 4 different functions now. Maybe abstract it to a method? So instead of 

```
if not self.voice_client:
    await message.channel.send(MusicBot.NO_VOICE_CLIENT_ERROR_MSG)
    return
```
you would do
```
if self.notify_if_voice_client_is_missing(message):
    return
```

Reply:
I was contemplating this but decided against it since the code snippet is so short. Thinking about it some more though, I do agree :)